app/controllers/items_controller.rb

```diff
@@ -347,6 +348,28 @@ def reindex
     Argo::Indexer.reindex_pid_remotely(@object.pid)
   end
 
+  def rights_list_for_apo(apo_id)
+    return Constants::REGISTRATION_RIGHTS_OPTIONS if apo_id.blank?
+
+    apo_object = Dor.find(apo_id)
+    default_opt = apo_object.default_rights
+    default_opt = 'citation-only' if default_opt == 'none'
```

Comment:
could this be:

```
default_opt = apo_object.default_rights || 'citation-only'
```

Reply:
@ndushay @aaron-collier Maybe I'm not reading this right or I have an incomplete understanding, but the "before" code I'm seeing in the snippet above would set `default_opt` to `'citation-only'` if already set to `'none'`. The suggested change only sets to `'citation-only'` if `default_opt` is falsey, but `'none'` is truthy. What am I missing?

Reply:
it was me missing stuff, maybe?

Reply:
I think your suggestion is safe, as 'none' isn't anything that is shown, and elsewhere in the code 'citation-only' is displayed as 'none. So I think this is just fine. Done.